### ISSUE TITLE
Req: list dependencies in readme.md

### ISSUE THREAD

**Issue by mikedan99** [witnessmenow/arduino-alarm-clock#1]
The vanilla Arduino IDE is missing many of the required libraries for the project to compile properly. I'm still chasing down at least one. I suggest you list the dependencies in the readme.md to make it easier to install all dependencies up front rather than compile-error-install-repeat.

I believe at least two of the required libraries are in the Arduino IDE Library Manager now, and at least one is not. I can list them if you'd want to add that to the project.

**Comment by maxint-rd** [comment]
FYI: I also had some libraries missing when trying out this example. 
These are the ones I needed to install (in Arduino IDE 1.6.10, ESP8266 core 2.4.1) before the code would compile.

Installed using library manager:
 - NTPClient by Fabrice Weinberg
 - ArduinoJson by Benoit Blanchon (NOTE: code uses version 5 instead of version 6!)
 - Json Streaming Parser by Daniel Eichhorn

Installed using GitHub downloaded zip file:
 - https://github.com/witnessmenow/arduino-google-maps-api
 - https://github.com/PaulStoffregen/Time 
 - https://github.com/JChristensen/Timezone   (some comment about "Modidified"?)

Then I got a compilation error (strcpy_F not declared in DateStrings.cpp:76:66). Apparently something was missing in my ESP8266 core v2.4.1, so I upgraded my core to the currently most recent version v2.5.0 

Finally I had acquire a proper API key and set some board settings before it would run all right (enable 3 MB spiffs)

(BTW: The TM1637 library was already installed, but my intention is to use my own TM16xx library which has some more features...)
